internal/build/lifecycle_execution.go

```diff
@@ -193,7 +193,7 @@ func (l *LifecycleExecution) Create(ctx context.Context, publish bool, dockerHos
 
 	opts := []PhaseConfigProviderOperation{
 		WithFlags(l.withLogLevel(flags...)...),
-		WithArgs(repoName),
+		WithArgs("-app", l.mountPaths.appDirName(), repoName),
```

Comment:
Does lifecycle expect it in the format `\appdir`, or in `c:\appdir`? If the later, we can use `mountPaths.appDir()` instead

Reply:
oh, i might need to revert 51887a3

Reply:
fixed in 9cc6613